examples/test_something.pp

```diff
@@ -4,3 +4,7 @@
   contact  => 'foo',
   location => 'unset',
 }
+
+banner { 'default':
```

Comment:
This could probably be it's own file under netdev/banner.pp ?

Reply:
Will get it moved now